3rdParty/fuerte/src/vst.cpp

```diff
@@ -187,23 +191,23 @@ void message::responseHeader(ResponseHeader const& header,
 
   // 3 - meta (not optional even if empty)
   builder.openObject();
-  if (!header.meta.empty()) {
-    for (auto const& item : header.meta) {
+  if (!header.meta().empty()) {
+    for (auto const& item : header.meta()) {
```

Comment:
do we need to handle `fu_content_type_key` separately here, or is that only required for requests, not responses?

Reply:
good point, the code is not used in arangodb